README.md

```diff
@@ -156,10 +156,8 @@ Please report big and small bugs by [opening an issue](https://github.com/eoscos
 
 <p align="center">
 	<a href="https://eoscostarica.io">
-		<img src="https://github.com/eoscostarica/eos-rate/raw/master/docs/eoscostarica-logo-black.png" width="300">
+		<img src="https://raw.githubusercontent.com/eoscostarica/.github/master/.github/workflows/images/eos-costa-rica-logo.png" width="300">
 	</a>
```

Comment:
you are missing the closing paragraph tag `</p>`

Reply:
fixed missing tag